test/src/bumper.spec.js

```diff
@@ -953,6 +1001,29 @@ describe('Bumper', () => {
       player.play();
     });
 
+    it('Should fire AD_PLAY_FAILED when the play promise failed', done => {
```

Comment:
whats the difference between this and the one in line 572?

Reply:
we have to 2 modes for bumper - with dedicated video element and with the playback video element. these 2 tests are in a different mode.

Reply:
ok then